packages/builder/src/components/design/settings/controls/DraggableList.svelte

```diff
@@ -33,17 +33,19 @@
   let anchors = {}
   let draggableItems = []
 
-  const buildDragable = items => {
-    return items.map(item => {
-      return {
-        id: listItemKey ? item[listItemKey] : generate(),
-        item,
-      }
-    })
+  const buildDraggable = items => {
+    return items
+      .map(item => {
+        return {
+          id: listItemKey ? item[listItemKey] : generate(),
+          item,
+        }
+      })
+      .filter(item => item.id)
```

Comment:
Filtering out nullish IDs, as otherwise this causes duplicate keys and crashes the page. They should never be null in the first place, but this is added protection since this is where the crash was occurring when using formula fields.